.travis.yml

```diff
@@ -14,7 +14,7 @@ matrix:
         - RUN=
 
     - os: linux
-      python: "3.3.6"
+      python: "3.6-dev"
```

Comment:
Should it be `3.6-dev` or `3.3.6-dev` ... ? May be there is a typo in https://github.com/travis-ci/travis-ci/issues/6437#issuecomment-241075722